phpcs.xml

```diff
@@ -19,5 +19,8 @@
     <file>test</file>
 
     <!-- Include all rules from the Laminas Coding Standard -->
-    <rule ref="LaminasCodingStandard"/>
+    <rule ref="LaminasCodingStandard">
+        <!-- Exclude rule -->
+        <exclude name="SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly.ReferenceViaFullyQualifiedName" />
```

Comment:
Why do we need to exclude this rule? 🤔

Reply:
For the `config/config.php` file because there we use fully qualified names.